packages/react/menu/src/Menu.tsx

```diff
@@ -715,18 +715,18 @@ type MenuSubTriggerPrimitive = Polymorphic.ForwardRefComponent<
 const MenuSubTrigger = React.forwardRef((props, forwardedRef) => {
   const context = useMenuContext(SUB_TRIGGER_NAME);
   const contentContext = useMenuContentContext(SUB_TRIGGER_NAME);
-  const openTimersRef = React.useRef<number[]>([]);
+  const contentRectRef = React.useRef<ClientRect | undefined>();
+  const openTimerRef = React.useRef<number | undefined>(undefined);
```

Comment:
I've managed to achieve this a different way using just one timer but checking if it's set before setting another one.

Reply:
Oh nice! 👍  could we be explicit with a `null` instead?

Reply:
I was being lazy because otherwise it moans when doing clearTimeout and I have to null-check it 😀 I'll update.